controllers/configuration_controller.go

```diff
@@ -491,8 +491,27 @@ func (r *ConfigurationReconciler) preCheck(ctx context.Context, configuration *v
 	}
 	meta.ConfigurationType = configurationType
 
+	// Check provider
```

Comment:
Why moving this block to here?

Reply:
Because we need use the credentials of provider to build backend.